openstack/rds/v3/backups/requests.go

```diff
@@ -41,3 +41,11 @@ func Update(c *golangsdk.ServiceClient, id string, opts UpdateOptsBuilder) (r Up
 	_, r.Err = c.Put(resourceURL(c, id), b, nil, reqOpt)
 	return
 }
+
+// GetBackupPolicy get the backup policy of the instance
+func GetBackupPolicy(client *golangsdk.ServiceClient, instanceID string) (r GetBackupPolicyResult) {
```

Comment:
please do not include the package name in the function name or struct name, which makes it look more simpler.

```suggestion
func Get(client *golangsdk.ServiceClient, instanceID string) (r GetResult) {
```

Reply:
ok,I have modified